webapp/TargetedMS/js/QCTrendPlotPanel.js

```diff
@@ -196,6 +185,47 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
         }
     },
 
+    getExperimentRunDetails: function (runId) {
+        var sql = "Select MIN(sf.AcquiredTime) AS StartDate,\n" +
+                "       MAX(sf.AcquiredTime) AS EndDate,\n" +
+                "       r.FileName,\n" +
+                "       sf.InstrumentSerialNumber,\n" +
+                "       sf.InstrumentId.model\n" +
+                "FROM targetedms.Replicate rep\n" +
+                "INNER JOIN targetedms.SampleFile sf ON rep.Id = sf.ReplicateId\n" +
+                "INNER JOIN targetedms.Runs r on r.Id = rep.RunId\n" +
+                "where rep.RunId ='" + runId + "'\n" +
+                "GROUP BY r.FileName, sf.InstrumentSerialNumber, sf.InstrumentId.model";
```

Comment:
This grouping approach won't work if the file in question has data from multiple instruments. We'll be grouping to separate them into different rows, but then we only use the first one (which may have nothing to do with this QC folder) in the success handler.

What are we doing with the serial number and model in the client side code right now?

Also, given the way that we generate the SQL behind the scenes for allFolders queries, we'd get better perf if you just do the SampleFile table in your FROM clause and use the lookup syntax to get the replicate and run columns.

Reply:
I will update the sql. 

Serial Number and model (instrument name) are used in the hover tooltip of experiment range highlighted region in QC Plots.

Reply:
Hmm... let's remove it from the tooltip and simplify the query so that it's no longer grouping or returning those columns. That will eliminate the problem of choosing the wrong instrument's data if there are multiple in the referenced Skyline document.

This will be better, though we should someday filter it for just the data that matches instrument in the QC folder.